nearit-ui/build.gradle

```diff
@@ -115,5 +115,5 @@ dependencies {
     compile 'com.android.support:appcompat-v7:26.1.0'
     compile 'com.android.support:support-v4:26.1.0'
     testCompile 'junit:junit:4.12'
-    provided 'it.near.sdk:nearit:[2.2.0, 2.2.2]'
+    provided 'it.near.sdk:nearit:[2.2.6, )'
```

Comment:
You are only guaranteed to not have the API broken if you stay on 2.2.x
With this you will also accept 2.3.x and 2.4.x